test/scaleTest.js

```diff
@@ -0,0 +1,36 @@
+/* global suite test assert */
+
+function targetTester (keyframeA, keyframeB, timing) {
+  var targetA = document.createElement('div');
+  document.body.appendChild(targetA);
+
+  var targetB = document.createElement('div');
+  document.body.appendChild(targetB);
+
+  var animation = targetA.animate(keyframeA, timing);
+  animation.currentTime = 1000;
+  var result = getComputedStyle(targetA).transform;
+
+  var animation = targetB.animate(keyframeB, timing);
+  animation.currentTime = 1000;
+  var expected = getComputedStyle(targetB).transform;
+
+  return result === expected;
```

Comment:
Let's remove targetA and targetB from the document now that we're done with them.